python/deltalake/__init__.py

```diff
@@ -36,3 +57,18 @@ def to_pyarrow_dataset(self) -> pyarrow.dataset.Dataset:
 
     def to_pyarrow_table(self) -> pyarrow.Table:
         return self.to_pyarrow_dataset().to_table()
+
+    def to_pyarrow_schema(self) -> pyarrow.Schema:
+        format = DeltaTableSchemaFormat.ARROW
+        delta_fields = [
+            DeltaTableField(
```

Comment:
Looks like the `DeltaTableField` is not being used at all? Why not just invoke `pyarrow.field` directly in this list comprehension loop?

Reply:
The `DeltaTableField` is used for extracting the Arrow data type in JSON format given by the `schema()` method.

The method `from_json()`(available in Rust) is not available in Python for Arrow (https://arrow.apache.org/docs/python/generated/pyarrow.Schema.html).

The main idea for the binding between Python and Rust was to use the `JSON` format for the data type binding. It avoids creating all the pyo3 structures for the binding of all the structures in the delta schema.